packages/block-editor/src/components/block-removal-warning-modal/index.js

```diff
@@ -17,10 +17,10 @@ import { store as blockEditorStore } from '../../store';
 import { unlock } from '../../lock-unlock';
 
 export function BlockRemovalWarningModal( { rules } ) {
```

Comment:
Do we still want to keep this component in "block-editor"?

Reply:
Ok with moving it, but then we still have the private actions, selectors and reducers for block removal warnings in the block editor package that are all the same family of APIs. I feel like it makes sense to keep those things together.

We could consider gradually moving it all across, but it'd mean changing how this feature integrates with the `removeBlocks` action pretty significantly. Will have a think about whether that's possible.

Reply:
Ok, let's keep it then :)